src/components/lists/TaskList.vue

```diff
@@ -879,8 +879,12 @@ export default {
         this.$t('tasks.fields.done_date'),
         this.$t('tasks.fields.last_comment_date')
       ]
-      if (!this.isAssets) {
-        headers.splice(4, 0, 'Frames')
+      if (this.isShots) {
+        if (this.isShots && !this.isPaperProduction) {
+          headers.splice(4, 0, this.$t('tasks.fields.frames'))
+        } else {
+          headers.splice(4, 0, this.$t('tasks.fields.drawings'))
+        }
```

Comment:
Duplidate condition `this.isShots`

```suggestion
      if (this.isShots) {
        const value = this.isPaperProduction
          ? this.$t('tasks.fields.drawings')
          : this.$t('tasks.fields.frames')
        headers.splice(4, 0, value)
      }
```